LeetCode/String/ZigZag_Conversion/HyeonJeong.py

```diff
@@ -0,0 +1,28 @@
+class Solution:
+    def convert(self, s: str, numRows: int) -> str:
+        nlist = []
+        alist = []
+        for i in range(numRows):
+            nlist += [i]
+            alist += [[]]
+        number = nlist + sorted(nlist, reverse=True)[1:-1]
+        # numRows에 따라 인덱스 순서의 반복 구간이 number에 리스트로 담김
+
+        n = 0
+        i = 0
+        while 1:
+            alist[number[i]] += s[n] # 해당하는 리스트 위치에 주어진 s를 넣음
+            n += 1
+            i += 1
+            if i == len(number): # 반복 구간을 반복될 수 있게 함
+                i = 0
+            if n == len(s): # s의 길이만큼 s의 문자들이 들어갈 수 있게 함
+                break
+
+        answer = ""
+        for string in alist:
+            for c in string:
+                answer += c
```

Comment:
```suggestion
        return join('', alist);
```
alist가 문자열의 배열이라면 이런식으로 표현할 수 있을 것 같아요!

Reply:
2차원 배열을 한번에 이을 수 있는게 뭐가 없나 고민했었는데, 1차원으로 만든 후 join()을 사용하겠습니다!